apps/desktop/desktop/src/main/java/bisq/desktop/main/content/authorized_role/moderator/BannedUserProfileTable.java

```diff
@@ -260,12 +346,49 @@ private static class ListItem {
 
             private final UserProfile userProfile;
             private final String userName;
+            private final String reporterUserProfileId;
+            private final String reporterUserName;
+            private final Optional<UserProfile> reporterUserProfile;
+            private final String banReason;
+
+            public static ListItem from(BannedUserProfileData data,
+                                        UserProfileService userProfileService,
+                                        ModeratorService moderatorService) {
+                return new ListItem(data, userProfileService, moderatorService);
+            }
 
-            private ListItem(BannedUserProfileData bannedUserProfileData) {
+            private ListItem(BannedUserProfileData bannedUserProfileData,
+                             UserProfileService userProfileService,
+                             ModeratorService moderatorService) {
                 this.bannedUserProfileData = bannedUserProfileData;
 
                 userProfile = bannedUserProfileData.getUserProfile();
                 userName = userProfile.getUserName();
+
+                String accusedUserProfileId = userProfile.getId();
+                BannedUserModeratorData moderatorData = moderatorService.getPersistableStore()
```

Comment:
We do not want to access the persistableStore from outside the service class. See https://github.com/bisq-network/bisq2/issues/3415

In that case its best to add a findBannedUserModeratorData(accusedUserProfileId) method which returns an optional.